### DISCUSSION TITLE
Are there any subFolders in nuxt3? How to configure them？

### DISCUSSION THREAD

**Discussion by gllinzbk1** [nuxt/nuxt#18803] [Questions]
thanks

**Comment by AloisSeckar** [comment]
Not sure if this is what you ask for, but:

There are several dedicated directories with special purpose, you can check them [in the docs](https://nuxt.com/docs/guide/directory-structure/nuxt). 

You can change them with `dir` option in Nuxt Config ([see here](https://nuxt.com/docs/api/configuration/nuxt-config#dir)), but it is not really recommended to do so, developers are rather encouraged to keep the default structure and best practices. It is then easier to understand the code, if you don't have to study configuration abnormalities first...

**Comment by gllinzbk1** [comment]
https://github.com/nuxt/docs/blob/master/en/api/configuration-generate.md
subFolders
Type: Boolean
Default: true
By default, running nuxt build && nuxt export(>= v2.13) or nuxt generate(<= v2.12) Nuxt.js will create a directory for each route & serve an index.html file.

Example:

-| dist/
---| index.html
---| about/
-----| index.html
---| products/
-----| item/
-------| index.html
When set to false, HTML files are generated according to the route path:

nuxt.config.js

export default {
  generate: {
    subFolders: false
  }
}
-| dist/
---| index.html
---| about.html
---| products/
-----| item.html

subFolders ,Can it be implemented in nuxt3?

**Comment by davidsandoz** [comment]
I believe this request is similar to unjs/nitro#607
